database/seeds/UsersTableSeeder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Faker\Factory as Faker;
+
+class UsersTableSeeder extends Seeder
```

Comment:
Better to extend this class from DatabaseSeeder and write parent method to seed. 
In this case we could choose the level of seeding, choosing a class in console. 
I.e. If we need only base installation, use base class. If we need also demo data, we use this class.